### PR TITLE
0.15.0 functions - repeat, clamp, member

### PR DESCRIPTION
**Pull request by javaduke**

### CODE REVIEW

**Comment by javaduke** [comment]
Addresses issue https://github.com/databricks/sjsonnet/issues/80

File: sjsonnet/src/sjsonnet/Std.scala [hunk 1]
```diff
@@ -353,6 +356,42 @@ object Std {
       }
       res
     },
+    builtin("member", "arr", "x"){ (ev, fs, arr: Val, x: Val) =>
+      val res = arr match {
+        case str: Val.Str =>
+          val secondArg = x match {
+            case Val.Str(value) => value
+            case n => Materializer(x)(ev).toString //The behavior of 'member' when first arg is string and second arg is not string is undefined - we do our best here
```
Comment (lihaoyi-databricks) [inline sjsonnet/src/sjsonnet/Std.scala:364]: It seems that `std.member` simply raises an error if the second argument `x` is not a string, at least when i tested it on jsonnet.org. Let's follow suite

Comment (javaduke) [reply sjsonnet/src/sjsonnet/Std.scala:364]: Agreed, will fix.

**Comment by lihaoyi-databricks** [comment]
@javaduke this looks good to me, can I merge it?

**Comment by javaduke** [comment]
Yes, please! 
Thank you sir!

**Pull request merged**